README.md

```diff
@@ -1 +1,52 @@
-# PolicyEngine app
+# Welcome :wave: !
+This is the official web app of the PolicyEngine project. <br/><br/>
+![PolicyEngineWebApp](https://user-images.githubusercontent.com/117248915/221730075-b368a1df-d0d4-455c-8768-329e1eb9318e.png)
+
+# Getting started
+
+## Prerequisites
+We recommend that you use the latest Node version 19. To easily manage your node versions, get the Node Version Manager [nvm](https://github.com/nvm-sh/nvm) and then do
+
+```
+nvm install 19 && nvm use 19
+```
+
+## Setup
+1. Clone the repo
+```
+https://github.com/PolicyEngine/policyengine-app.git
+```
+2. Install dependencies
+
+```
+npm install
+```
+3. Start a server on localhost to see your changes
+```
+npm start
+```
+Now you're ready to start developing!
+
+To test your changes, do
+```
+npm test
+```
+
+# Contributing
+If you're new to the project, go to Issues and search for good first issues `label: "good first issue"`.
+
+Reach out to @nikhilwoodruff and @MaxGhenis to be added as a collaborator so that you can make new PRs on the repo.
```

Comment:
Anyone can make a PR.
```suggestion
Reach out to @nikhilwoodruff and @MaxGhenis to be added as a collaborator so that you can assign yourself to issues.
```